testing/it_sidecar/client/noop.setup

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+# This is a no-op setup script that simulates the K8Setup script output
+echo "FORWARD foo"
```

Comment:
The forward line per [it_sidecar.go#L290](https://github.com/adobe/rules_gitops/blob/eaef1a6189b48bd4cbac6fd77a8cb5a7dcac9586/testing/it_sidecar/it_sidecar.go#L290) is

```go
fmt.Printf("FORWARD %s:%d:%d\n", serviceName, port.Remote, port.Local)
```